lib/server.js

```diff
@@ -132,15 +133,23 @@ function createServer(options, clients, name) {
         name: common.METRIC_REQUEST_COUNTER,
         help: 'count of Muskie requests completed'
     });
-    // A histogram to track the time to first byte.
+    /*
+     * A log-linear histogram to track the time to first byte.
+     * Track values between 10^(-4) and 10^(4) seconds.
+     */
     options.collector.histogram({
         name: common.METRIC_LATENCY_HISTOGRAM,
-        help: 'time-to-first-byte of Muskie requests'
+        help: 'time-to-first-byte of Muskie requests',
+        buckets: artedi.logLinearBuckets(10, -4, 3, 4)
     });
-    // A histogram to track the time it took to fully process each HTTP request.
+    /*
+     * A log-linear histogram to track the time it took to fully process each
+     * HTTP request. Track values between 10^(-4) and 10^(4) seconds.
```

Comment:
I believe this should be ms? Though, if we are fixing this and wiping out the data, I wonder if it would make sense to change to the prometheus-standard of using  seconds here. What do you think? In other words the two options I see are:

 * update the comment and make a note that this value is `http_request_time_ms` in milliseconds
 * change to `http_request_time_seconds` which would be the correct thing to do according to the prometheus best practices.

Doing the latter would actually mean we don't *have* to wipe all the existing data if we didn't want to. Since we could just update queries to use the new metric name instead.

Reply:
@joshwilsdon  - That is correct, it should be `ms`, I updated the comment and the histogram buckets accordingly. For the second point, updating the metrics to seconds instead of ms, it would be great to follow prometheus-standard here. However, I suggest keeping the values in ms. The reason is that I am trying to avoid breaking the existing prometheus queries ops use.

Reply:
Perhaps I should mention that I removed `http_request_time_ms` because it is not used. See comments in the Jira ticket [MANTA-4388](https://jira.joyent.us/browse/MANTA-4388).